test/hotspot/jtreg/vmTestbase/gc/gctests/LoadUnloadGC2/LoadUnloadGC2.java

```diff
@@ -49,10 +52,8 @@ public void run() {
                 stresser.start(500000);
                 try {
                         while (stresser.iteration()) {
-                                GarbageProducer garbageProducer = new GeneratedClassProducer();
                                 log.info("Iteration: " + stresser.getIteration());
-                                GarbageUtils.eatMemory(stresser, garbageProducer, 0);
-                                garbageProducer = null;
+                                WhiteBox.getWhiteBox().fullGC();
```

Comment:
We need to create some classes to unload them. Are they created?

Reply:
I missed that part. I will correct it.